middleware/healthcheck/healthcheck_test.go

```diff
@@ -51,13 +55,17 @@ func Test_HealthCheck_Default(t *testing.T) {
 	app := fiber.New()
 	app.Get("/livez", NewHealthChecker())
 	app.Get("/readyz", NewHealthChecker())
+	app.Get(DefaultStartupEndpoint, NewHealthChecker())
```

Comment:
I think that maybe to have everything padronized the other paths should use the default variable too or this path should use a string path just like the others (I would personally prefer the first option, just didn't remembered it when I wrote the middleware).

What do you think @ReneWerner87?

Reply:
@kirankumar-grootan yeah pls use the default var everywhere in ths test

Reply:
@ReneWerner87  changed from string to use the const in test cases